frontend/Components/Login.js

```diff
@@ -0,0 +1,210 @@
+            </View>
+            {Platform.OS == 'web' &&
+            <View  style={Styles.imgContainer}>
+            <Image
+                source={require('../assets/images/themeImage.png')}
+                style={Styles.themeImage}
+            />
+            </View>}
+        </View>
+    );
+};
+
+const Styles = StyleSheet.create({
+    device: {
+        flexDirection: 'row',
+        backgroundColor: '#dffaef',
+        alignItems: 'flex-start'
+    },
+
+    container: {
+        marginHorizontal: '5%',
+        flex: 2,
+        justifyContent: 'flex-start',
+        backgroundColor: "#dffaef",
+        height: '100%'
+    },
+
+    logo: {
+        height: '15%',
+        width: '54%',
+        resizeMode: 'contain'
+    },
+
+    imgContainer: {
+        marginTop: "5%",
+        marginHorizontal: '5%',
+        flex: 4,
+        justifyContent: 'center',
+        backgroundColor: "#dffaef",
+        height: '100%'
+    },
+
+    themeImage: {
+        resizeMode: 'contain',
+        height: '100%',
+        borderRadius: 38,
+    },
+
+    title: {
+        fontSize: 36,
+        fontWeight: 'bold',
+    },
+
+    titleContainer: {
+        width: '100%',
+        height: '20%',
+        justifyContent: 'center',
+        paddingLeft: Platform.OS !== 'web' ? '10%' : '40%',
+        marginTop: '3%'
+    },
+
+    textBoxContainer: {
+        backgroundColor: 'white',
+        height: Platform.OS === 'web' ? '15%' : '8%',
+        marginVertical: '5%',
+        borderRadius: 10,
+        padding: '2%',
+    },
+
+    textBox: {
+        paddingHorizontal: '3%',
+        fontSize: 16,
+    },
+
+    button: {
+        marginVertical: '5%',
+        backgroundColor: '#0066F9',
+        borderRadius: 20,
+        alignItems: 'center',
+        justifyContent: 'center',
+        height: Platform.OS === 'web' ? '13%' : '8%'
+    },
+
+    buttonText: {
+        color: 'white',
+        fontSize: 20,
+        fontWeight: 'bold',
+    },
+
+    farLink: {
+        marginVertical: '4%',
+        marginLeft: '5%',
+        width: '50%'
+    },
+
+    lastLine: {
+        marginVertical: '4%',
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+
+    smallLink: {
+        fontWeight: 'bold',
+        textDecorationLine: 'underline',
+        fontSize: 14
+    },
+
+    inLineLink: {
+    },
+
+    mediumLink: {
+        fontWeight: 'bold',
+        textDecorationLine: 'underline',
+        fontSize: 16
+    }
+
+});
+
+const logoPath = require('../assets/images/LOGO.png'); 
+
+function MakeTextInput(props) {
+    const [value, onChangeText] = React.useState('');
+
+    return (
+        <View style={Styles.textBoxContainer}>
+            <Text style={{fontSize:16,margin:3}}>{props.type}</Text>
+            <TextInput
+                style={Styles.textBox}
+                placeholder={'Enter your ' + props.type}
+                placeholderTextColor={'gray'}
+                secureTextEntry={props.type==='Password'}
```

Comment:
One other thing i would like to suggest is to avoid having specific behavior based on the props like in this case, because it violates the open/closed principle since if we had more types of inputs we would have to more if statements, instead we just do this
```suggestion
                {...props}
```
and when we use the component we add `secureTextEntry={true}` like this `<MakeTextInput name="Password" secureTextEntry={true}/>`.
I understand the reason for it could be to avoid duplication, but we could do so by just creating another wrapper component `PasswordInput` and add it to the above code and every time we want a password input we use it instead. 
However this just a one line of code so i find it unnecessary.